src/services/redisSessionStorage.ts

```diff
@@ -83,7 +84,9 @@ export default class RedisSessionStorage extends RedisStorageUtils
       user.fiscal_code
     );
 
-    const removeOtherUserSessionsPromise = this.removeOtherUserSessions(user);
+    const removeOtherUserSessionsPromise = this.allowMultipleSessions
```

Comment:
```suggestion
    const removeOtherUserSessionsOrNopPromise = this.allowMultipleSessions
```

(otherwise we'd have a `removeOtherUserSessionsPromise` later in the code that maybe does nothing which may be surprising)

Reply:
Fixed in b7ed01d